Guide-GroundSystem.md

```diff
@@ -35,6 +35,7 @@ Installing and running cFS Ground System on Ubuntu:
 1. ```sudo apt-get install python3-zmq```
 1. ```sudo apt-get install libcanberra-gtk-module```
 1. ```cd Subsystems/cmdUtil/ && make```
+1. ```cd ../..```
```

Comment:
These two lines can be replaced w/ `make -C Subsystems/cmdUtil`, no need to actually change directories.